app/src/androidTest/java/net/skyscanner/backpack/docs/GenerateScreenshots1.kt

```diff
@@ -0,0 +1,92 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.docs
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatDelegate
+import androidx.compose.ui.test.junit4.AndroidComposeTestRule
+import androidx.test.rule.ActivityTestRule
+import net.skyscanner.backpack.demo.BpkBaseActivity
+import net.skyscanner.backpack.demo.meta.StoriesRepository
+import net.skyscanner.backpack.demo.meta.Story
+import net.skyscanner.backpack.demo.ui.BpkScaffold
+import net.skyscanner.backpack.demo.ui.StoryScreen
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+
+// todo: rename it when we all the stories migrated
+@RunWith(Parameterized::class)
+open class GenerateScreenshots1(
+  private val story: Story,
+) {
+
+  companion object {
+
+    @JvmStatic
+    @Parameterized.Parameters(name = "{0} Screenshot")
+    fun data(): List<Story> = StoriesRepository.getInstance().screenshotStories()
+
+  }
+
+  @get:Rule
+  var activityRule = ActivityTestRule(BpkBaseActivity::class.java, true, false)
```

Comment:
did you try and see if we can switch to `ActivityScenario` for this now?

Reply:
Nope, not as a part of this ticket for now